<record reset="true">
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542674567992</miliseconds>
  <timestamp>11/20/2018 00:42:47:992</timestamp>
  <thread>0</thread>
  <class>Connection</class>
  <method>newSocket</method>
  <message>Connection refused: connect (com.microstrategy.webapi.MSTRWebAPIException)</message>
  <exception>com.microstrategy.webapi.MSTRWebAPIException: Connection refused: connect&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.Connection.newSocket(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.CreateConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.AddDSSXMLClusterNode(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.Connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebClusterAdminImpl.connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper$ServerConnectionsUpdateTask.run(Unknown Source)&#x0D;&#x0A;&#x09;at java.util.TimerThread.mainLoop(Timer.java:555)&#x0D;&#x0A;&#x09;at java.util.TimerThread.run(Timer.java:505)&#x0D;&#x0A;Caused by: java.net.ConnectException: Connection refused: connect&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.connect0(Native Method)&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)&#x0D;&#x0A;&#x09;at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)&#x0D;&#x0A;&#x09;at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:589)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:538)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:434)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:211)&#x0D;&#x0A;&#x09;... 12 more&#x0D;&#x0A;</exception>
</record>
<record>
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542674568055</miliseconds>
  <timestamp>11/20/2018 00:42:48:055</timestamp>
  <thread>0</thread>
  <class>CDSSXMLClusterNode</class>
  <method>CreateConnection</method>
  <message>EXCEPTION</message>
  <parameters>
    <parameter>JUNE-PC</parameter>
    <parameter>34952</parameter>
  </parameters>
</record>
<record>
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542674568055</miliseconds>
  <timestamp>11/20/2018 00:42:48:055</timestamp>
  <thread>0</thread>
  <class>CDSSXMLClusterNode</class>
  <method>CreateConnection</method>
  <message>The connection failed. Possible causes are a) Server is not running, b) Server is loading project, c)  Network problem, or d) Server reached maximum connection limit. (com.microstrategy.webapi.MSTRWebAPIException)</message>
  <exception>com.microstrategy.webapi.MSTRWebAPIException: The connection failed. Possible causes are a) Server is not running, b) Server is loading project, c)  Network problem, or d) Server reached maximum connection limit.&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.CreateConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.AddDSSXMLClusterNode(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.Connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebClusterAdminImpl.connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper$ServerConnectionsUpdateTask.run(Unknown Source)&#x0D;&#x0A;&#x09;at java.util.TimerThread.mainLoop(Timer.java:555)&#x0D;&#x0A;&#x09;at java.util.TimerThread.run(Timer.java:505)&#x0D;&#x0A;Caused by: com.microstrategy.webapi.MSTRWebAPIException: Connection refused: connect&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.Connection.newSocket(Unknown Source)&#x0D;&#x0A;&#x09;... 11 more&#x0D;&#x0A;Caused by: java.net.ConnectException: Connection refused: connect&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.connect0(Native Method)&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)&#x0D;&#x0A;&#x09;at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)&#x0D;&#x0A;&#x09;at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:589)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:538)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:434)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:211)&#x0D;&#x0A;&#x09;... 12 more&#x0D;&#x0A;</exception>
</record>
<record>
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542675175026</miliseconds>
  <timestamp>11/20/2018 00:52:55:026</timestamp>
  <thread>1</thread>
  <class>CDSSXMLServerSessionImpl</class>
  <method>CreateSessionEx</method>
  <message>(로그인 실패.)</message>
  <exception>com.microstrategy.webapi.MSTRWebAPIException: (로그인 실패.)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLServerSessionImpl.handleError(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLServerSessionImpl.CreateSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLServerSessionImpl.CreateSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLServerSessionImpl.CreateSessionEx(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebIServerSessionImpl.createSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebIServerSessionImpl.createSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebIServerSessionImpl.createNewSessionID(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebSessionInfoImpl.getSessionID(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebIServerSessionImpl.reconnect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.WebAppSessionManagerImpl.createSessionUsingES(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.WebAppSessionManagerImpl.validateSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.WebAppSessionManagerImpl.validateSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.ServletEventHandlerImpl.setupIServerSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.ServletEventHandlerImpl.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.AdminServletEventHandler.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.beans.GenericEventHandler.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.ServletEventHandlerImpl.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.AdminServletEventHandler.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.beans.AbstractWebComponent.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.controller.GenericWebAppController.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.controller.GenericWebAppController.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.servlets.MainServlet.doGet(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.servlets.MainServlet.doPost(Unknown Source)&#x0D;&#x0A;&#x09;at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)&#x0D;&#x0A;&#x09;at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)&#x0D;&#x0A;&#x09;at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)&#x0D;&#x0A;&#x09;at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)&#x0D;&#x0A;&#x09;at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)&#x0D;&#x0A;&#x09;at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:616)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)&#x0D;&#x0A;&#x09;at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:521)&#x0D;&#x0A;&#x09;at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1096)&#x0D;&#x0A;&#x09;at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:674)&#x0D;&#x0A;&#x09;at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.doRun(AprEndpoint.java:2500)&#x0D;&#x0A;&#x09;at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.run(AprEndpoint.java:2489)&#x0D;&#x0A;&#x09;at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)&#x0D;&#x0A;&#x09;at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)&#x0D;&#x0A;&#x09;at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)&#x0D;&#x0A;&#x09;at java.lang.Thread.run(Thread.java:745)&#x0D;&#x0A;</exception>
  <parameters>
    <parameter>JUNE-PC</parameter>
    <parameter>0</parameter>
    <parameter></parameter>
    <parameter></parameter>
    <parameter>1</parameter>
    <parameter>admin</parameter>
  </parameters>
</record>
<record reset="true">
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542675609450</miliseconds>
  <timestamp>11/20/2018 01:00:09:466</timestamp>
  <thread>0</thread>
  <class>Connection</class>
  <method>newSocket</method>
  <message>Connection refused: connect (com.microstrategy.webapi.MSTRWebAPIException)</message>
  <exception>com.microstrategy.webapi.MSTRWebAPIException: Connection refused: connect&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.Connection.newSocket(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.CreateConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.AddDSSXMLClusterNode(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.Connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebClusterAdminImpl.connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper$ServerConnectionsUpdateTask.run(Unknown Source)&#x0D;&#x0A;&#x09;at java.util.TimerThread.mainLoop(Timer.java:555)&#x0D;&#x0A;&#x09;at java.util.TimerThread.run(Timer.java:505)&#x0D;&#x0A;Caused by: java.net.ConnectException: Connection refused: connect&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.connect0(Native Method)&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)&#x0D;&#x0A;&#x09;at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)&#x0D;&#x0A;&#x09;at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:589)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:538)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:434)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:211)&#x0D;&#x0A;&#x09;... 12 more&#x0D;&#x0A;</exception>
</record>
<record>
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542675609528</miliseconds>
  <timestamp>11/20/2018 01:00:09:528</timestamp>
  <thread>0</thread>
  <class>CDSSXMLClusterNode</class>
  <method>CreateConnection</method>
  <message>EXCEPTION</message>
  <parameters>
    <parameter>JUNE-PC</parameter>
    <parameter>34952</parameter>
  </parameters>
</record>
<record>
  <package>com.microstrategy.webapi</package>
  <level>SEVERE</level>
  <miliseconds>1542675609528</miliseconds>
  <timestamp>11/20/2018 01:00:09:528</timestamp>
  <thread>0</thread>
  <class>CDSSXMLClusterNode</class>
  <method>CreateConnection</method>
  <message>The connection failed. Possible causes are a) Server is not running, b) Server is loading project, c)  Network problem, or d) Server reached maximum connection limit. (com.microstrategy.webapi.MSTRWebAPIException)</message>
  <exception>com.microstrategy.webapi.MSTRWebAPIException: The connection failed. Possible causes are a) Server is not running, b) Server is loading project, c)  Network problem, or d) Server reached maximum connection limit.&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.CreateConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLClusterNode.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.AddDSSXMLClusterNode(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.InitConnection(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.CDSSXMLAdmin.Connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.objects.WebClusterAdminImpl.connect(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper.connectIServers(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.admin.beans.AdminServersHelper$ServerConnectionsUpdateTask.run(Unknown Source)&#x0D;&#x0A;&#x09;at java.util.TimerThread.mainLoop(Timer.java:555)&#x0D;&#x0A;&#x09;at java.util.TimerThread.run(Timer.java:505)&#x0D;&#x0A;Caused by: com.microstrategy.webapi.MSTRWebAPIException: Connection refused: connect&#x0D;&#x0A;&#x09;at com.microstrategy.webapi.Connection.newSocket(Unknown Source)&#x0D;&#x0A;&#x09;... 11 more&#x0D;&#x0A;Caused by: java.net.ConnectException: Connection refused: connect&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.connect0(Native Method)&#x0D;&#x0A;&#x09;at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)&#x0D;&#x0A;&#x09;at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)&#x0D;&#x0A;&#x09;at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)&#x0D;&#x0A;&#x09;at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:589)&#x0D;&#x0A;&#x09;at java.net.Socket.connect(Socket.java:538)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:434)&#x0D;&#x0A;&#x09;at java.net.Socket.&lt;init&gt;(Socket.java:211)&#x0D;&#x0A;&#x09;... 12 more&#x0D;&#x0A;</exception>
</record>
<record>
  <package>com.microstrategy.web.app.beans</package>
  <level>SEVERE</level>
  <miliseconds>1542675693262</miliseconds>
  <timestamp>11/20/2018 01:01:33:262</timestamp>
  <thread>1</thread>
  <class>WebAppSessionManagerImpl</class>
  <method>createSessionUsingES</method>
  <message>Invalid Login Mode</message>
  <exception>com.microstrategy.web.app.WebAppException: Invalid Login Mode&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.WebAppSessionManagerImpl.createSessionUsingES(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.WebAppSessionManagerImpl.validateSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.WebAppSessionManagerImpl.validateSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.ServletEventHandlerImpl.setupIServerSession(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.ServletEventHandlerImpl.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.AdminServletEventHandler.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.beans.GenericEventHandler.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.ServletEventHandlerImpl.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.app.beans.AdminServletEventHandler.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.beans.AbstractWebComponent.handleRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.controller.GenericWebAppController.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.controller.GenericWebAppController.processRequest(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.servlets.MainServlet.doGet(Unknown Source)&#x0D;&#x0A;&#x09;at com.microstrategy.web.servlets.MainServlet.doPost(Unknown Source)&#x0D;&#x0A;&#x09;at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)&#x0D;&#x0A;&#x09;at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)&#x0D;&#x0A;&#x09;at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)&#x0D;&#x0A;&#x09;at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)&#x0D;&#x0A;&#x09;at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)&#x0D;&#x0A;&#x09;at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:616)&#x0D;&#x0A;&#x09;at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)&#x0D;&#x0A;&#x09;at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:521)&#x0D;&#x0A;&#x09;at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1096)&#x0D;&#x0A;&#x09;at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:674)&#x0D;&#x0A;&#x09;at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.doRun(AprEndpoint.java:2500)&#x0D;&#x0A;&#x09;at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.run(AprEndpoint.java:2489)&#x0D;&#x0A;&#x09;at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)&#x0D;&#x0A;&#x09;at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)&#x0D;&#x0A;&#x09;at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)&#x0D;&#x0A;&#x09;at java.lang.Thread.run(Thread.java:745)&#x0D;&#x0A;</exception>
</record>
